Base/usr/share/man/man1/Eyes.md

```diff
@@ -1,6 +1,4 @@
-## Name
```

Comment:
Why remove the name header? It's part of the man page format, why should it be removed for GUI apps?

Reply:
I didn't even notice, yeah that should definitely be kept.

Reply:
I thought it looked nicer but if that's the preferred format I'll change it back.